diff --git a/node_modules/react-native-splash-view/SplashView.podspec b/node_modules/react-native-splash-view/SplashView.podspec
index b9d4df2..86d2262 100644
--- a/node_modules/react-native-splash-view/SplashView.podspec
+++ b/node_modules/react-native-splash-view/SplashView.podspec
@@ -19,7 +19,9 @@ Pod::Spec.new do |s|
   s.source_files = "ios/**/*.{h,m,mm,cpp,swift}"
   s.preserve_paths = 'ios/SplashView-Bridging-Header.h'
 
-  s.dependency "React-RCTFabric"
+  if ENV['RCT_NEW_ARCH_ENABLED'] == '1'
+    s.dependency "React-RCTFabric"
+  end
   
   s.swift_version = '5.0'
   s.requires_arc = true
